libkbfs/keybase_service_base.go

```diff
@@ -649,6 +649,8 @@ func (k *KeybaseServiceBase) GetTLFCryptKeys(ctx context.Context,
 		return keybase1.GetTLFCryptKeysRes{}, err
 	}
 
+	ctx = context.WithValue(ctx, CtxNoAddToFavoriteKey{}, true)
```

Comment:
`true` seems weird, since setting it to `false` doesn't do the opposite.  Maybe just set it to `struct{}{}`?